docs/dvl/intro.md

```diff
@@ -15,4 +15,13 @@ To facilitate the generation of distributed validator keys amongst remote users
 
 ## Getting started
 
-For more information on running charon in a UI friendly way through the DV Launchpad, take a look at our [Quickstart Guides](docs/int/quickstart/index.md). 
\ No newline at end of file
+For more information on running charon in a UI friendly way through the DV Launchpad, take a look at our [Quickstart Guides](docs/int/quickstart/index.md). 
```

Comment:
I can't comment this but on line 14 do you also want to change it to beta/holesky?

Reply:
There's goerli all over the site right now. Moving it all to holesky is something we can do in the new year and its own PR rather than making multi-network language here.